packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -152,6 +152,7 @@ public ReactImageView(
     mDraweeControllerBuilder = draweeControllerBuilder;
     mGlobalImageLoadListener = globalImageLoadListener;
     mCallerContext = callerContext;
+    setLegacyVisibilityHandlingEnabled(true);
```

Comment:
Can you add a comment here explaining why this is necessary / when we can move away from it?

Reply:
I've taken [the fix from the Fresco repo](https://github.com/facebook/fresco/issues/1445#issuecomment-315763953). I couldn’t see any information on the Fresco repo about a longer term solution. Could you check with @oprisnik (he works at facebook), please? He indicated [that work on Drawee is frozen](https://github.com/facebook/fresco/pull/2391#issuecomment-783379351)

Reply:
That would be fine for context. Something like 

> Work around Android bug where ImageView visibility is not propagated to the Drawable, so you have to manually update visibility. Will be resolved once we move to VitoView.